crates/rlp/Cargo.toml

```diff
@@ -33,3 +33,8 @@ derive = ["dep:alloy-rlp-derive"]
 
 arrayvec = ["dep:arrayvec"]
 smol_str = ["dep:smol_str"]
+
+[[example]]
+name = "enum"
+crate-type = ["staticlib"]
+path = "../../examples/enum.rs"
```

Comment:
maybe examples should be in `crates/rlp`?

Reply:
This is fine